src/templates/sidebar.html

```
@@ -16,6 +16,28 @@ <h3 class="hidden-xs">
+  {% if request.user.is_staff %}
+    <div class="nav list-group hidden-xs sidebar-menu" id="lg-menu-staff">
+      <a class="list-group-item {% if '/approvals/' in request.path_info %}active{% endif %}"
+         href="{% url 'quests:approvals' %}">
+        <i class="fa fa-shield fa-fw fa-rotate-180"></i>&nbsp;
+        <span class="visible-sm-inline visible-xs-inline visible-md-inline">Approvals</span>
+        <span class="hidden-sm hidden-xs hidden-md">Quest Approvals</span>
+        <span id="approvals_badge" class="badge alert-danger list-group-item-badge "></span>
+      </a>
+    </div>
+
+    <div class="nav list-group hidden-xs sidebar-menu" id="lg-menu-staff">
```

HTMLHint (1.5.0)

[error] The id value [ lg-menu-staff ] must be unique.

(id-unique)